### PR TITLE
Fail build on bad yarn locks

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Adds a guard that CI can catch if our yarn lock gets out of synch with package.json

Preemptive sol'n for https://github.com/ca-cwds/react-wood-duck/issues/32 without having to resort to pre-commit hooks

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
Closed #34 in favor of this one. (Using correct GH acct to propose merge)

**Comment by edgibbs** [comment]
Can one of the admins verify this patch?

File: .editorconfig [hunk 1]
```diff
@@ -7,3 +7,6 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[Dockerfile]
+indent_size = 4
```
Comment (edgibbs) [inline .editorconfig]: Quick question for indent size 4 for Dockerfiles?

Comment (eckdanny-osi) [reply .editorconfig]: groovy-lang style guide docs from Apache are all like that. Seemed like the a good idea to anchor to that

**Pull request merged**